_i18n/en.yml

```
@@ -60,6 +60,13 @@ contribute:
   develop:
     title: "Develop"
     text: "You can program in Java? Help test pull requests or submit new features or bug fixes on GitHub."
+  communicate:
+    title: "Communicate"
+    text: "You draft catchy tweets, interesting blogs, or wonderful graphics? Help spread the word."
+  design:
+    title: "Design"
+    text: "You turn complex requests in understandable user stories or wireframes? Help tackle these issues."
+  donate: "Or, if you can't contribute in any of the ways above, you can make a <a href="https://opencollective.com/antennapod" target="_blank">donation</a>."
```

Should be "_listed_ above"
```suggestion
  donate: "Or, if you can't contribute in any of the ways mentioned above, you can make a <a href="https://opencollective.com/antennapod" target="_blank">donation</a>."
```